test/benchmarks/mongoBench/suites/singleBench.js

```diff
@@ -13,8 +13,30 @@ const {
 
 function makeSingleBench(suite) {
   suite
+    .benchmark('returnDocument', benchmark =>
+      benchmark
+        .taskSize(1.531e-3) // One tweet is 1,531 bytes or 0.001531 MB
+        .setup(makeLoadJSON('tweet.json'))
+        .task(async function () {
```

Comment:
Any reason why such a small task would run for 6.4 seconds?

```
{
info: { test_name: 'returnDocument', tags: [ 'js-bson' ] },
metrics: [ { name: 'megabytes_per_second', value: 6744.493919146219 } ]
}
```

See [here](https://parsley.mongodb.com/evergreen/mongo_node_driver_next_performance_tests_run_spec_benchmark_tests_node_18_server_6.0_patch_833eaa41c5f379268be46408282b0c97323ea1bc_66e063e5822f450007766577_24_09_10_15_21_11/0/task?bookmarks=0,564) for the full performance file data.

Reply:
I misread mb/s as ms, my bad! looks good to go for me.